### PR TITLE
chore(core): rename "hidden wallet" -> "passphrase wallet"

### LINKED ISSUE #4057: Rename "Hidden wallet" to "Passphrase wallet" everywhere

### PR DESCRIPTION
**Pull request by ibz**
Fixes #4057

### CODE REVIEW

**Review by obrusvit — COMMENTED** [review]
Unfortunately it does not fit on some screens of TS3 :crying_cat_face: 
image

How about model One? If I checked correctly, there is one string containing "hidden wallet" in `legacy/firmware/config.c`, line `602`. It's for sure not a reason to make new FW version but maybe we can make the change now so that it's not forgotten in the future?

**Comment by Hannsek** [comment]
Don't we have a marquee in the header here<

**Comment by Hannsek** [comment]
And also not a big problem as almost no one will ever see this screen.

**Comment by ibz** [comment]
> Don't we have a marquee in the header here<
> 
> Regarding T1. If it it a long hanging fruit (10 mins max), then let's do it, otherwise let's not do it.

I can confirm that the header is a marquee and the text is readable. I double checked.

I tried updating the string in Model One and checking how it looks in the emulator and in that case the text becomes unreadable - due to the same issue: it being in the header and too long - but in that case it is *not* a marquee...

image

**Comment by Hannsek** [comment]
Let's not do it then.

File: core/translations/es.json
Comment (obrusvit) [inline core/translations/es.json:1]: Please check the UI for spanish version.

File: core/translations/fr.json
Comment (obrusvit) [inline core/translations/fr.json:1]: Please check the UI for french version.

**Review by obrusvit — COMMENTED** [review]
Please check the UI in other langs and post screenshots.

File: core/translations/es.json
Comment (ibz) [reply core/translations/es.json:1]: image

It *almost* fits...

File: core/translations/fr.json
Comment (ibz) [reply core/translations/fr.json:1]: It fits!

image

**Review by obrusvit — APPROVED** [review]

File: core/translations/fr.json
Comment (prusnak) [reply core/translations/fr.json:1]: but the inverted question mark is a bug, no?

Comment (Hannsek) [reply core/translations/fr.json:1]: Yeah, there should be `'`.

Comment (obrusvit) [reply core/translations/fr.json:1]: We probably miss the `'` in here https://github.com/trezor/trezor-firmware/blob/33f5d4733065ac631d61ff458a5db32c81641124/core/tools/codegen/foreign_chars.py#L36 
That's why it fallbacks to non-printable char :question:

Comment (prusnak) [reply core/translations/fr.json:1]: maybe just change the translation to use normal ASCII `'`?

**Comment by bosomt** [comment]
**QA OK**

**Info:**
 - Suite version: desktop 24.7.3 (19fd64f5b2ab1bc265821e34d4c4b9c910864d0e)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuite/24.7.3 Chrome/118.0.5993.159 Electron/27.3.8 Safari/537.36
 - OS: MacIntel
 - Screen: 1512x982
 - Device: Trezor **T3T1** 2.8.1 regular (revision 33f5d4733065ac631d61ff458a5db32c81641124)
 - Device: Trezor **T2B1** 2.8.1 regular (revision 33f5d4733065ac631d61ff458a5db32c81641124)
 - Transport: BridgeTransport 2.0.33

**Pull request merged**